symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserDetailMapper.java

```diff
@@ -0,0 +1,60 @@
+package com.symphony.bdk.core.service.user;
+
+import com.symphony.bdk.gen.api.model.UserAttributes;
+import com.symphony.bdk.gen.api.model.UserDetail;
+import com.symphony.bdk.gen.api.model.V2UserAttributes;
+import com.symphony.bdk.gen.api.model.V2UserDetail;
+
+class UserDetailMapper {
```

Comment:
Could we use https://mapstruct.org/ ?

Reply:
Updated but I need your review on pom file because the build on CircleCI failed on maven-compiler!

Reply:
Is failing locally as well?

Reply:
it failed on local as well